reco_utils/recommender/deeprec/DataModel/ImplicitCF.py

```diff
@@ -0,0 +1,213 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import random
+import numpy as np
+import pandas as pd
+import scipy.sparse as sp
+import time
+from reco_utils.common.constants import (
+    DEFAULT_ITEM_COL,
+    DEFAULT_USER_COL,
+    DEFAULT_RATING_COL,
+    DEFAULT_PREDICTION_COL,
+)
+
+
+class ImplicitCF(object):
+    """Dataset class for LightGCN"""
```

Comment:
You should give a better summary of what is ImplicitCF instead of "Dataset class for LightGCN"